src/App.vue

```diff
@@ -726,17 +730,43 @@ onBeforeUnmount(() => {
 const currentTopBarHeightPixels = computed(() => `${widgetStore.currentTopBarHeightPixels}px`)
 const currentBottomBarHeightPixels = computed(() => `${widgetStore.currentBottomBarHeightPixels}px`)
 
-const showDiscoveryDialog = ref(false)
 const preventAutoSearch = useStorage('cockpit-prevent-auto-vehicle-discovery-dialog', false)
 
-onMounted(() => {
+const runElectronStartup = async (): Promise<void> => {
+  if (isElectron()) {
+    interfaceStore.enqueueDialog({
+      id: 'UpdateNotification',
+      component: UpdateNotification,
+      props: {},
+    })
+    // Wait 5 seconds to check if we're connected to a vehicle
+    setTimeout(() => {
+      if (vehicleStore.isVehicleOnline) return
+      interfaceStore.enqueueDialog({
+        id: 'VehicleDiscoveryDialog',
+        component: VehicleDiscoveryDialog,
+        props: {},
+      })
+      if (interfaceStore.dialogQueue[0].id === 'VehicleDiscoveryDialog') {
+        interfaceStore.openNextDialogOnQueue()
+      }
+    }, 5000)
+  }
```

Comment:
Can you move this function to after the `onMounted`? This way git knows its a change on the `onMounted` function and not a complete rewrite of those lines. 

It's important to prevent those changes on the positions of code blocks when not needed.